src/app/counter/Button/button.css

```diff
@@ -0,0 +1,11 @@
+.dec-button {
```

Comment:
might be good to make a button class and then have inc-button and dec-button just change the color